modules/microshift-install-rhel-image-mode-build-image.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-build-image_{context}"]
+= Building the bootc image
+
+Build your {op-system-base-full} that contains {microshift-short} as a bootable container image by using a a Containerfile.
```

Comment:
```suggestion
Build your {op-system-base-full} that contains {microshift-short} as a bootc image by using a a Containerfile.
```

Reply:
We have been advised to use "bootc image" instead of "bootable containers".

Reply:
```suggestion
Build your {op-system-base-full} that contains {microshift-short} as a bootc image by using a Containerfile.
```